### PR TITLE
Fix typo in type name

### PR DESCRIPTION
**Pull request by BasThomas**
👀

### CODE REVIEW

File: Pods/Pods.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -1545,39 +1545,39 @@
 /* Begin PBXFileReference section */
 		0034CEB0AE79BBC3F3651389899C926E /* IGListWorkingRangeHandler.h */ = {isa = PBXFileReference; includeInIndex = 1; lastKnownFileType = sourcecode.c.h; name = IGListWorkingRangeHandler.h; path = Source/Internal/IGListWorkingRangeHandler.h; sourceTree = "<group>"; };
 		0053D94F32AEB2F8AF7075C38C6FFF39 /* CGImage+LRUCachable.swift */ = {isa = PBXFileReference; includeInIndex = 1; lastKnownFileType = sourcecode.swift; name = "CGImage+LRUCachable.swift"; path = "StyledText/CGImage+LRUCachable.swift"; sourceTree = "<group>"; };
-		00652DD8B12BC3EC3A931F198EF26387 /* xcode.min.css */ = {isa = PBXFileReference; includeInIndex = 1; name = xcode.min.css; path = Pod/Assets/styles/xcode.min.css; sourceTree = "<group>"; };
```
Comment (BasThomas) [inline Pods/Pods.xcodeproj/project.pbxproj]: No idea why it did all this stuff 🤷‍♂️

**Comment by rnystrom** [comment]
Must be missing an optional file field with new networking stack because the files won’t load.

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Comment by BasThomas** [comment]
I don't get it. 😛

**Comment by Sherlouk** [comment]
typeo 🤷‍♂️ 
(_sorry_)

**Comment by rnystrom** [comment]
Talking to myself, I can’t load the files on GitHawk.

GitHawk Upload by rnystrom

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Comment by BasThomas** [comment]
Ah I'll hunt.

**Comment by rnystrom** [comment]
oops i broke pods again

**Comment by rnystrom** [comment]
@BasThomas mind backing out the pods changes? Then I can merge this.

**Comment by BasThomas** [comment]
Yeah I’ll pick up on Tuesday!

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Comment by BasThomas** [comment]
Oops, I did something wrong. To be honest I don't get what I need to do here to fix this? 😕

**Comment by rnystrom** [comment]
You can just rebase and when it says there are conflicts run `pod install` again. We just want to update the local pod tho so it picks up the file name change.

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Comment by rnystrom** [comment]
Cleaning up & closing this out, low hanging fruit for someone that wants to start contributing

**Pull request closed without merging**